src/App.js

```diff
@@ -25,7 +25,10 @@ class App {
 
 		this.model = wwtController.getModel();
 
-		this.revisionPopup = new RevisionPopupWidget();
+		const $overlay = wwtController.getOverlay();
+		this.revisionPopup = new RevisionPopupWidget( {
+			$overlay
```

Comment:
The RevisionPopupWidget doesn't actually use $overlay (as @mooeypoo says below), so there's no point in passing it in. (This should probably also be `$overlay: $overlay`, if the widget parent class is fixed to accept an overlay.)